src/mem/cache/prefetch/Prefetcher.py

```
@@ -204,7 +204,7 @@ class XSStridePrefetcher(QueuedPrefetcher):
     fuzzy_stride_matching = Param.Bool(False, "Match stride with fuzzy condition")
     short_stride_thres = Param.Unsigned(512, "Ignore short strides when there are long strides (Bytes)")
     stride_dyn_depth = Param.Bool(False, "Dynamic depth of stride table")
-    stride_entries = Param.MemorySize("32", "Stride Entries")
+    stride_entries = Param.MemorySize("10", "Stride Entries")
```

Ruff (0.14.5)

`Param` may be undefined, or defined from star imports

(F405)